### ISSUE TITLE
Delete the concept of a circle because it duplicates the concept of a role?

### ISSUE THREAD

**Issue by bernardmariechiquet** [holacracyone/Holacracy-Constitution#254]
As defined in [article 1.3](https://github.com/holacracyone/Holacracy-Constitution/blob/master/Holacracy-Constitution.md#13-circles) a circle is a role. This is a major shift and conceptual simplification of the concepts embedded into the constitution and this is great!. Now the two concepts are the same, so I wonder why we would still need two different names which brings confusion both for the new comers in Holacracy and also the already experienced 4.1 ones. I would propose to remove the circle concept and the reason behind is that it creates confusion. Then we would have Roles, Role Leader, Role Supporter, Sub-Roles, Super-Role and may remove Circle (Role) and Circle Lead (Role Lead).
I'd love to get some reaction, may be there are good reasons for keeping *circle* I don't see.

**Comment by bernardmariechiquet** [comment]
I was just reflecting, may be it's easier to remove Role and keep only Circle, and a Circle may contain or not other Circles. Circle Lead, Circle Supporter, Sub-Circle and Super-Circle. Wouldn't it simplify a lot. An holarchy being a Circle containing Circles containing themselves Circles....

**Comment by brianjrobertson** [comment]
My own sense is that this would be far more confusing, especially to novices.  It also loses that the two words do refer to different perspectives: A "Role" is the outside view of the holon; a "Circle" is the inside view of the holon.  I'll leave the issue open for a bit to allow debate before I go with my judgment and drop it.

**Comment by cassus** [comment]
I observed novices calling all roles circles anyway (probably inspired by the GlassFrog graphic).
My understanding is that since there is no practical difference between role and circle it's fine whichever we use in a novice group, no need to remove one of the concepts as they refer to the same thing from a novices perspective.

**Comment by bernardmariechiquet** [comment]
> My own sense is that this would be far more confusing, especially to novices. It also loses that the two words do refer to different perspectives: A "Role" is the outside view of the holon; a "Circle" is the inside view of the holon. I'll leave the issue open for a bit to allow debate before I go with my judgment and drop it.

It would be easier to refer one word for the Role/Circle concept and add the level of holarchy. Back to #258 for instance. Your answer : 

> That's right; the broader Super-Circle lead is only assigning people as an overall Circle Lead (or Circle Lead supporters) to the overall Circle, not to specific Roles within the Circle; the Circle Leads of the Circle itself are assigning people into the Roles within the Circle, whether or not those people are also Circle Leads (or Circle Lead supporters).

Let me rephrase it with Role and Level concepts versus Cuircle and Role concepts : 
**Roles level n are assigned by (Role Leads of) Roles level n+1.** 
The language would be far easier. All this has already been defined by holon theory. And if language is easier, communication and learning too.

**Comment by julianeroell** [comment]
For me, the two words are very much needed, if just for efficiency: From the position of a _Role_, I need to be able to reference my surrounding / higher system (currently: the "Circle"), and I need that reference to have a different name than that which I use for my peers (other "Roles").

**Comment by brianjrobertson** [comment]
I was already feeling strongly negative on this one intuitively, and @martinaroell's comment is quite compelling beyond just my intuition; I'm dropping this one.

**Comment by bernardmariechiquet** [comment]
> For me, the two words are very much needed, if just for efficiency: From the position of a Role, I need to be able to reference my surrounding / higher system (currently: the "Circle"), and I need that reference to have a different name than that which I use for my peers (other "Roles").

@martinaroell @brianjrobertson The issue is that the current mental model re role/circle is completely changed from the current 4.1 we are used to. For instance, from a position of a Role inside the Circle,  take the Role Supporter ("on the Circle") concept defined at section 1.2.3,  **it** is doing work within the Circle like a peer, then how could you reference it from a position of a Role inside the Circle? 

For me the language has to change, I will keep moving on this one. As of today, the language used do not allow to communicate properly all the new concepts introduced in 5.0.
